invertedai_cpp/invertedai/session.h

```diff
@@ -36,6 +39,13 @@ class Session {
   double max_backoff = 0; // No max backoff by default, 0 signifies no limit
   double jitter_factor = 0.5;
 
+  const char *iai_logger_path_char = std::getenv("IAI_LOGGER_PATH");
+  bool is_log_path_null = iai_logger_path_char == NULL;
+  const std::string iai_logger_path = !is_log_path_null ? iai_logger_path_char : "./";
+  const bool is_logging = (!is_log_path_null) && (std::filesystem::is_directory(std::filesystem::path(iai_logger_path)));
```

Comment:
I think we should attempt to create this directory if it doesn't exist, and only report a warning if that fails.

Reply:
Fixed